### ISSUE TITLE
UI GKEv2 cluster creation - Input Fields do not look enabled

### ISSUE THREAD

**Issue by sowmyav27** [rancher/dashboard#3774]
**On 2.6-head commit id: `723f747`**

- GKEv2 cluster --> Edit or add a new cluster
- In the node pool section the nodepool name, and other input fields do _not_ look enabled. Looks like they are greyed out or disabled.

Can we have the input fields another colour ?

<img width="1387" alt="Screen Shot 2021-08-10 at 7 35 11 PM" src="https://user-images.githubusercontent.com/26032343/128960875-154e9fad-7e55-42f1-bc29-f0905f16e2dd.png">

**Comment by brudnak** [comment]
# Reproduction Environment

| Component                | Version / Type                                     |
| ------------------------ | -------------------------------------------------- |
| Rancher version          | v2.6-99b5fbcfba33b5af75f6b229855c25ad96b43dc8-head |
| Installation option      | Docker install                                     |
| Docker version           | 20.10.7, build 20.10.7-0ubuntu1~20.04.2            |
| Helm version             | v2.16.8-rancher1                                   |
| Infrastructure           | Linode                                             |
| Browser type             | Google Chrome                                      |
| Browser version          | 94.0.4606.81 (Official Build) (x86_64)             |
| Rancher Theme Preference | Dark                                               |

**Reproduction steps:**

1. Starting from the Rancher homepage `/dashboard/home`
2. Click the `hamburger menu` -> `Cluster Management` -> `Create` -> `Google GKE`
3. Observe the various input fields

**There are  two colors used for various available input fields.** In first table presented below you can see that one of the colors used for available input fields is 4.9% darker. Two screen shots also included below labeled `A1` & `A2`

| Background Input Color Hex | RGB             | Lightness |
| -------------------------- | --------------- | --------- |
| #26282f                    | rgb(38, 40, 47) | 16.7%     |
| #1b1c21                    | rgb(27, 28, 33) | 11.8%     |
  
### Additional Info


**Table of Inputs & Their Background Colors**
>This is on the Google GKE creation form (Ember in Vue)
<table>
  <tbody>
    <tr>
      <th>Section</th>
      <th align="center">Input & Background Color</th>
    </tr>
    <tr>
      <td>First input</td>
      <td align="center">Cluster Name - [#26282f] (lighter)</td>
    </tr>
    <tr>
      <td>Labels & Annotations</td>
      <td>
        <ul>
          <li>Key - [#26282f] (lighter)</li>
          <li>Value - [#26282f] (lighter)</li>
        </ul>
      </td>
    </tr>
    <tr>
      <td>Google Account Access</td>
    <td>
        <ul>
          <li>Google Project ID - [#26282f] (lighter)</li>
          <li>Cloud Credentials - [#1b1c21] (darker)</li>
        </ul>
      </td>
    </tr>
    <tr>
      <td>Cluster Location</td>
      <td align="center">Zone - [#26282f] (lighter)</td>
    </tr>
    <tr>
    <td>Cluster Options</td>
    <td>
        <ul>
          <li>Kubernetes Version - [##1b1c21] (darker)</li>
          <li>Container Address Range - [#26282f] (lighter)</li>
          <li>Network - [#26282f] (lighter)</li>
          <li>Node Subnet - [#26282f] (lighter)</li>
          <li>Pod secondary Range Name - [#26282f] (lighter)</li>
          <li>Pod Secondary CIDR Block - [#26282f] (lighter)</li>
          <li>Services Secondary Range Name - [#26282f] (lighter)</li>
          <li>Service Secondary CIDR Block - [#26282f] (lighter)</li>
          <li>Master IPV4 CIDR Block- [#26282f] (lighter)</li>
        </ul>
      </td>
    </tr>
    <td>Additional Options</td>
    <td>
        <ul>
          <li>Logging Service - [##1b1c21] (darker)</li>
          <li>Monitoring Service - [##1b1c21] (darker)</li>
          <li>Maintenance Window - [##1b1c21] (darker)</li>
          <li>Cluster Description - [#26282f] (lighter)</li>
          <li>Cluster Labels Key - [#26282f] (lighter)</li>
          <li>Cluster Labels Value - [#26282f] (lighter)</li>
        </ul>
      </td>
    </tr>
    <td>Node Pools</td>
    <td>
        <ul>
          <li>Image Type - [##1b1c21] (darker)</li>
          <li>Machine Type - [##1b1c21] (darker)</li>
          <li>Root disk type - [##1b1c21] (darker)</li>
          <li>Root Disk Size - [##1b1c21] (darker)</li>
          <li>Local SSD disks - [##1b1c21] (darker)</li>
          <li>Effect - [##1b1c21] (darker)</li>
          <li>Effect Key - [##1b1c21] (darker)</li>
          <li>Effect Value - [##1b1c21] (darker)</li>
          <li>Group Name - [##1b1c21] (darker)</li>
          <li>Initial Node Count - [##1b1c21] (darker)</li>
          <li>Max Pod Per Node - [##1b1c21] (darker)</li>
        </ul>
      </td>
    </tr>
  </tbody>
</table>

### Screenshots

![01-3374-a1](https://user-images.githubusercontent.com/40877149/138339286-ec6fbe3f-cc1f-4889-aa58-9dbd74ae4a3e.png)

![02-3374-a2](https://user-images.githubusercontent.com/40877149/138339335-06a34938-99be-45a7-96b5-ca1bed2b1786.png)

-----

**Comment by brudnak** [comment]
# ✅ PASSED

## Reproduction Environment

#### Not required.

<hr>

## Validation Environment



| Component                        | Version / Type                                                                     |
| -------------------------------- | ---------------------------------------------------------------------------------- |
| Rancher version                  | v2.7-e077f670ab72023102d36725ed1e2e7e56dbcdd4-head                                 |
| Rancher commit link              | https://github.com/rancher/rancher/commit/e077f670ab72023102d36725ed1e2e7e56dbcdd4 |
| Installation option              | Helm (high availability)                                                           |
| If Helm Chart k8s cluster        | v1.24.7+k3s1                                                                       |
| Cert Details                     | Let's Encrypt / `traefik`                                                          |
| k3s ha external db               | Aurora MySQL 5.7.mysql_aurora.2.10.2                                               |
| Helm version                     | v2.16.8-rancher2                                                                   |
| Downstream cluster type          | not applicable                                                                     |
| Downstream K8s version           | not applicable                                                                     |
| Authentication providers enabled | local                                                                              |
| Logged in user role              | Administrator                                                                      |
| Browser type                     | Google Chrome                                                                      |
| Browser version                  | 108.0.5359.98 (Official Build) (x86_64)                                            |


<details>
    <summary>🚨 Additional Reproduction Setup Details: Click to Expand</summary>

>K3s install command for node one. Replace values in <>

```shell
curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=<K3S_VERSION> sh -s - server --token=SECRET --datastore-endpoint=<DB_ENDPOINT> --tls-san <TLD> --node-external-ip <IP>
```

>K3s install command for node two. Replace values in <>. Main difference here is the --token

```shell
curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=<K3S_VERSION> sh -s - server --token=<TOKEN_FROM_NODE_ONE> --datastore-endpoint=<DB_ENDPOINT> --tls-san <TLD> --node-external-ip <IP>
```


</details>

## Validation steps

1. Start the process of creating a GKE cluster
2. Fill out the cloud credential form to get to the node pool section
3. Once at the node pool section observe the fields

## Additional Info

## RESULTS

#### ✅ Expected
For the node pool inputs to match the color of other text input fields, and not appear to be grayed out or disabled

#### ✅ Actual
The node pool text input section for GKE clusters now looks as expected and matches other text input fields
